src/pages/__tests__/Index.test.tsx

```
@@ -129,6 +130,12 @@ vi.mock("@/components/TruthGateFlow", () => ({
 
 vi.mock("@/components/UploadZone", () => ({
   default: (props: Record<string, unknown>) => {
+    useEffect(() => {
```

ESLint

[error] React Hook "useEffect" is called in function "default" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

(react-hooks/rules-of-hooks)